samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.h

```diff
@@ -15,6 +15,7 @@ enum temperature { ONOFF_TEMP = 0x01, LEVEL_TEMP, CTL_TEMP, IGNORE_TEMP};
 
 struct generic_onoff_state {
 	u8_t onoff;
+	u8_t tmp;
```

Comment:
Is it necessary to put temporary variable into this structure?

Reply:
Kindly consider same explanation as above.